modules/run-vault/run-vault

```diff
@@ -104,6 +104,19 @@ function assert_is_installed {
   fi
 }
 
+function vault_version_at_least {
+  local readonly config_path=$1
+  local readonly ui_config=$2
+  local readonly VAULT_VERSION=$(/usr/local/bin/vault -v|awk '{print $2}'|tr -d v)
```

Comment:
I had to squint to figure out what this did. To help future maintainers, could you extract this into its own (one-line) named method (e.g., `get_vault_version`) and add a comment explaining what Vault's version output looks like and how this method strips the leading v.